fake/tests/determinism.rs

```diff
@@ -272,6 +272,42 @@ use fake::faker::phone_number::raw::*;
 check_determinism! { l10d CellNumber; String, fake_cell_number_en, fake_cell_number_fr, fake_cell_number_cn, fake_cell_number_tw, fake_cell_number_jp }
 check_determinism! { l10d PhoneNumber; String, fake_phone_number_en, fake_phone_number_fr, fake_phone_number_cn, fake_phone_number_tw, fake_phone_number_jp }
 
+#[cfg(feature = "geo")]
```

Comment:
this should be `geo-types` , so that it is possible to test just the types, when other structs from `geo` are added.